backend/src/main/java/com/javahelp/backend/domain/user/register/RegisterUserResponse.java

```diff
@@ -0,0 +1,70 @@
+package com.javahelp.backend.domain.user.register;
+
+import com.javahelp.model.user.User;
+import com.javahelp.model.user.UserPassword;
+
+public class RegisterUserResponse {
+
+    private User user;
+    private UserPassword password;
+    private String error;
+
+    /**
+     * For bundle up info of the result of this successful registration
+     * @param user the newly created {@link User}
+     * @param password the newly created {@link UserPassword}
+     */
+    public RegisterUserResponse(User user, UserPassword password){
```

Comment:
These constructors should probably not be public. You want to be able to interact with this class publicly, but the only time a new instance should be created, is by the interactor, which is in the same package. These should keep default visibility.